.MANIFEST.in

```diff
@@ -0,0 +1,4 @@
+include README.md
```

Comment:
This is still referencing the README.md.

Reply:
Shoot. I'll fix that. 😅

Reply:
https://github.com/andersinno/cmsplugin-google-maps-ai/pull/2